docs/users/coaches.md

```diff
@@ -23,32 +23,33 @@ You can add new coaches using the _Add New Coach_ option, and edit them by click
 
 **Coach Status**: To distinguish coaches who are no longer working with the school, you can assign them the status of _exited_. Feedback and messages from these coaches are preserved, but they no longer have access to coach-specific features.
 
-## Assigning coaches to students
+## Assigning coaches to cohorts of a course
 
-Once a coach has been added to a school, they can immediately access all communities. However, to review submissions from students, they must be assigned to students in one of two ways:
+Once a coach has been added to a school, they can immediately access all communities. However, to review submissions from students, they must be assigned to a cohort of a course.
 
 ### Course Coaches
```

Comment:
We don't have a concept of course coach anymore; This should be updated to reflect the cohort change